pkg/printers/internalversion/printers_test.go

```diff
@@ -6601,7 +6601,7 @@ func TestPrintIPAddressList(t *testing.T) {
 	// Columns: Name, ParentRef, Age
 	expected := []metav1.TableRow{
 		{Cells: []interface{}{"192.168.2.2", "myresource.mygroup/mynamespace/myname", "10y"}},
-		{Cells: []interface{}{"2001:db8::2", "myresource2.mygroup2/mynamespace2/myname2", "5y1d"}},
+		{Cells: []interface{}{"2001:db8::2", "myresource2.mygroup2/mynamespace2/myname2", "5y2d"}},
```

Comment:
ha, I think this is failing because of leap day ... we need to adjust this so we don't fail every time the date range gains or loses a leap day

Reply:
yes. The current test used -5 years.

Reply:
change the timestamp of the objects to a zero timestamp so this column becomes `<unknown>`

we then need to cherry-pick this to 1.27+ which are now failing this test :-/

Reply:
Updated.

Another tricky way is use `time.Now().AddDate(0, 0, -5*365)` instead of `time.Now().AddDate(-5, 0, 0)`.